components/director/internal/domain/formationconstraint/operators/utils.go

```diff
@@ -68,3 +71,12 @@ func RetrieveNotificationStatusReportPointer(ctx context.Context, notificationSt
 
 	return notificationStatusReport, nil
 }
+
+func contains(s []string, e string) bool {
```

Comment:
If this is only used in `destination_creator_operator.go`, maybe `slices.Contains` can be used? It basically does the same.